tests/forms_tests/field_tests/test_multiplechoicefield.py

```diff
@@ -68,3 +68,8 @@ def test_multiplechoicefield_changed(self):
         self.assertFalse(f.has_changed([2, 1], ['1', '2']))
         self.assertTrue(f.has_changed([1, 2], ['1']))
         self.assertTrue(f.has_changed([1, 2], ['1', '3']))
+
+    def test_disabled_has_changed(self):
+        f = MultipleChoiceField(choices=[('1', 'One'), ('2', 'Two')],
```

Comment:
Use a single line. Our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) allows lines up to 119 characters if it helps readability.